Updated Proximal Policy Optimization example for Keras v3

lpizzinidev opened the pull request:
Updates the Proximal Policy Optimization example for Keras v3.
The example will be tf-only as the `GradientTape` function is used.

fchollet reviewed (commented):
Thanks for the PR! The code looks good. Please add the generated files.

lpizzinidev commented:
@fchollet Thanks for reviewing 👍
I added the generated files and updated the script as well (was working in Colab but not with `autogen.py`) by migrating to Gymnasium (OpenAI gym is deprecated) and fixing other issues.

fchollet reviewed (approved):
LGTM, thank you!
